lab_1_keywords_tfidf/start.py

```diff
@@ -18,7 +27,25 @@ def main() -> None:
         idf = load(file)
     with open("assets/corpus_frequencies.json", "r", encoding="utf-8") as file:
         corpus_freqs = load(file)
-    result = None
+
+    cleaned_tokens = clean_and_tokenize(target_text)
+
+    removed_stop_words = remove_stop_words(cleaned_tokens, stop_words)
+
+    calculated_frequencies = calculate_frequencies(removed_stop_words)
+
+    tf = calculate_tf(calculated_frequencies)
+
+    tfidf = calculate_tfidf(tf, idf)
+
+    top_frequencies = get_top_n(tfidf, 10)
+
+    result = top_frequencies
+
+    if not (cleaned_tokens, removed_stop_words, calculated_frequencies, tf, tfidf):
```

Comment:
why are you checking it here? For example: you put `cleaned_tokens` to some functions, got result and in the end you want to check if is it None? May be you should check it earlier?